src/Gaufrette/Adapter/GoogleCloudStorage.php

```diff
@@ -351,4 +362,30 @@ private function getObjectData($path, $options = array())
             return false;
         }
     }
+
+    /**
+     * @param $filename
+     * @param null $expires
+     * @return string
+     * @throws \Google_Auth_Exception
+     */
+    public function getUrl($filename, $expires = null)
+    {
+        $expiresTime = time() + $expires;
+        $to_sign = ("GET\n\n\n" . $expiresTime . "\n/" . $this->bucket . '/' . $filename);
```

Comment:
There is no rule for that. Please see another line from this file (for example: 
https://github.com/modernfactory/Gaufrette/blob/master/src/Gaufrette/Adapter/GoogleCloudStorage.php#L231) or this line from another file: https://github.com/KnpLabs/Gaufrette/blob/master/src/Gaufrette/Adapter/Ftp.php#L166